airflow/providers/google/cloud/hooks/cloud_sql.py

```diff
@@ -15,10 +15,18 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-"""This module contains a Google Cloud SQL Hook."""
+"""This module contains a Google Cloud SQL Hook.
+
+.. spelling:word-list::
```

Comment:
It is better to add these words to this list: https://github.com/apache/airflow/blob/main/docs/spelling_wordlist.txt

Reply:
I didn't know about this file. Thank you!

Reply:
I didn't know about this file. Thank you!